RIOT-OS/RIOT
# tools: whitespacecheck: use ignore file to easily add expressions

### Contribution description
The `whitespacecheck` script in `dist/tools/` currently ignores directory structures that have `/vendor/` in the name.

This PR adds an `ignore_files.txt` to easily add more expressions without touching the script.

Needed by #14654, because it introduces `fixdep.c` from the linux sources without adjusting the source code style in order to reduce the patch diff size.


### Testing procedure
* whitespacechecks should run as expected in murdock and travis.
* You can also locally run `./dist/tools/whitrespacecheck/check.sh master` and add whitespaces in different file. Then you can ignore the modified files by adding them to the `ignore_files.txt`

### Issues/PRs references
